combat-trainer.lic

```diff
@@ -423,6 +438,7 @@ class LootProcess
         tried_loot ||= !matches.empty?
         matches.each { |_| items_to_loot.push(item); }
       end
+    items_to_loot = items_to_loot - @box_nouns if @box_loot_limit and @current_box_count >= @box_loot_limit
```

Comment:
You could instead add `.reject { |item| ... }`  to the function chain on line 433.

Reply:
I think that would be better. trying this:
      .reject { |item| @box_nounts.include?(item) && @box_loot_limit && @current_box_count >= @box_loot_limit }

Will test.